lib/ZumoOta/src/FileManager.cpp

```diff
@@ -71,9 +75,18 @@ FileManager::~FileManager()
    
 bool FileManager::init()
 {
+    if (true == Logging::getInstance().registerSink(&gLogSinkSerial))

```

Comment:
see former comment.

Reply:
It will be taken care of by the next commit.